features/sensor_readings_api/identify_via_address.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Identify Sensors by I2C Address
+  As a member of the service team,
+  I want my sensor readings to include an I2C Address
+  So the story.board can distinguish two similar sensors based on this address
+
+  Background:
+    Given I send and accept JSON
+
+  Scenario Outline: Receive sensor reading with an I2C address
+    Given I have a sensor with a I2C address <address>
+    When I send a POST request to "/sensor_readings" with the following:
+    """
+    {
+    "calibrated_value": 47,
+    "uncalibrated_value": 11,
+    "sensor": {
+        "address": <address>
+      }
+    }
+    """
+    Then the response status should be "201"
+    And now the sensor has a new sensor reading in the database
+
+    Examples:
+      | address   |
+      | "4711"    | # either a decimal number
+      | "0xAF"    | # or a hex number
+      | "0xbC"    | # hex is case-insensitive
```

Comment:
@drjakob @kabel42 this is the new behaviour. You can send a JSON to ```/sensor_readings ```with a lookup by address. It doesn't matter whether this is a decimal string or a hex code.